manager/templates/default/element/tv/renders/input/listbox-multiple.tpl

```diff
@@ -30,22 +28,12 @@ Ext.onReady(function() {
         ,valueField: "value"
         ,resizable: true
         ,allowBlank: {if $params.allowBlank == 1 || $params.allowBlank == 'true'}true{else}false{/if}
-
         {if $params.title|default},title: '{$params.title|default}'{/if}
-        {if $params.listWidth|default},listWidth: {$params.listWidth|default}{/if}
-        ,maxHeight: {if $params.maxHeight|default}{$params.maxHeight|default}{else}300{/if}
-        {if $params.typeAhead|default}
```

Comment:
This check was wrong, because the value is a string with 'true' or 'false' in almost every case. So typeAhead is always set to true.

It should be checked like in allowBlank:

```
{if $params.typeAhead == 1 || $params.typeAhead == 'true'}
```

Reply:
It should be the same issue in listbox-single